### DISCUSSION TITLE
Is it possible to see what tags an item has?

### DISCUSSION THREAD

**Discussion by ChristopherHaws** [shedaniel/RoughlyEnoughItems#1270] [Q & A] [answered]
I am not sure what tags different items have and was wondering if it is possible to get them to show up in the item tooltips for easier discovery?

**Comment by shedaniel** [comment, marked as answer]
Download a mod like KubeJS, then you can press shift or ctrl (I don't remember) to show the tags

**Reply by ChristopherHaws** [reply]
I just got around to trying this out. KubeJS didn't add what your are talking about however I did find another mod that does exactly what you are talking about just in case anyone else is interested: https://www.curseforge.com/minecraft/mc-mods/better-tips

**Reply by MaxNeedsSnacks** [reply]
KubeJS does add this feature, but it's been disabled by default because pack authors never disable this stuff for production and we've gotten some **really** frustrated players in our server saying they can't see the item's tooltip anymore because of the tag list lmao
